api/cluster/resource/templater.go

```diff
@@ -152,10 +174,34 @@ func (t *InferenceServiceTemplater) PatchInferenceServiceSpec(orig *kservev1beta
 	}
 	orig.ObjectMeta.Annotations = utils.MergeMaps(utils.ExcludeKeys(orig.ObjectMeta.Annotations, configAnnotationKeys), annotations)
 	orig.Spec.Predictor = createPredictorSpec(modelService, config)
+	if err != nil {
```

Comment:
We can remove this since createPredictorSpec does not return error

Reply:
Same comment as above! Thanks again!